src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -576,7 +576,7 @@ private static function initPlaceholderFormatters()
                 return Helper::formatTime(time() - $bar->getStartTime());
             },
             'remaining' => function (ProgressBar $bar) {
-                if (!$bar->getMaxSteps()) {
+                if (null === $bar->getMaxSteps()) {
                     throw new \LogicException('Unable to display the remaining time if the maximum number of steps is not set.');
                 }
```

Comment:
thanks for mentioning it, I've just made sure it doesn't turn into a division by zero.